utils/installer/install.ps1

```diff
@@ -0,0 +1,155 @@
+#Set Variable to master is not set differently
+if ($null -eq $LVBRANCH) {
+    $LVBRANCH = "master"
+}
+
+#set -o nounset # error when referencing undefined variable
+$ErrorActionPreference = "Stop" # exit when command fails
+
+function mkdirondemand($path) {
+	if ((Test-Path $path) -eq $false) {
+		mkdir $path
+	}
+}
+
+function commandinstalled($cmd){
+	try { Get-Command $cmd > $null }
+	catch { return $false }
+	return $true
+}
+
+function moveoldlvim {
+	Write-Output "Not installing LunarVim"
+	Write-Output "Please move your $HOME\.local\share\lunarvim folder before installing"
+	exit
+}
+
+function asktoinstallpip {
+	Write-Output "Please install pip3 before continuing with install"
+	exit
+}
+
+function installnode {
+	if ((commandinstalled("winget.exe")) -eq $false) {
+		Write-Output "winget is not installed, but is the only package manager currently supported."
+		Write-Output "node is not installed. Please install it manually"
+		exit
+	}
+	winget.exe install "OpenJS.NodeJSLTS"
+}
+
+function asktoinstallnode {
+	Write-Output "node not found"
+	$answer = Read-Host "Would you like to install node now (y/n)" -Confirm
+	if($answer.ToLower() -eq 'y') {
+		installnode
+	}
+}
+
+function installpynvim {
+	Write-Output "Installing pynvim..."
+	pip3.exe install pynvim --user
+}
+
+function installpacker {
+	git clone https://github.com/wbthomason/packer.nvim ${HOME}/.local/share/lunarvim/site/pack/packer/start/packer.nvim
+}
+
+function modifyhomevar($config) {
+	
+	Copy-Item $config $config".save"
+	(Get-Content -Path $config) |
+		ForEach-Object {$_ -Replace 'os.getenv "HOME"', 'os.getenv "HOMEDRIVE" .. os.getenv "HOMEPATH"'} |
+			Set-Content -Path $config
+}
+
+function cloneconfig($arg) {
+	Write-Output "Cloning LunarVim configuration"
+	mkdirondemand("$HOME/.local/share/lunarvim")
+	
+	if ($arg.Contains("--testing")) {
+		Copy-Item -Recurse "${pwd}" "$HOME/.local/share/lunarvim/lvim"
+	} else {
+		Write-Output "lvbranch=$LVBRANCH"
+		git clone --branch "$LVBRANCH" https://github.com/ChristianChiarulli/lunarvim.git "$HOME/.local/share/lunarvim/lvim"
```

Comment:
can you use the new repo instead of this 🥺 
`https://github.com/LunarVim/LunarVim.git`

Reply:
@abzcoding Changed it. Thanks for pointing it out.